addons/binding/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/handler/MaxCubeBridgeHandler.java

```diff
@@ -614,6 +621,9 @@ private void readLines(String terminator) throws IOException {
         while (cont) {
             String raw = reader.readLine();
             if (raw != null) {
+                if (backup > 0) {
+                    bk.buildMsg(raw);
```

Comment:
yes, the parsing in the handler is actually done bit late for the backup utility.
It made it rather complex to workaround that.
Hence indeed a minor parsing was the most simple workaround to avoid more complexity in the handler. (which is already way to complex for what it is doing)